blog/admin.py

```diff
@@ -0,0 +1,20 @@
+from django.contrib import admin
+from django.contrib.auth.models import Group
+from .models import Post, Commentary, User
```

Comment:
Prefer using absolute imports instead of relative ones to be not dependent on the project structure. Edit it in all relevant places